src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAY_LENGTH = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[ARRAY_LENGTH];
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
```

Comment:
Returning `null` from a method is a bad practice. Make sure that the `getDefaultFigure` method in `FigureSupplier` does not return `null`. If necessary, handle the last possible option in the `default` case of a `switch` statement.